frontend/src/interfaces/RoomInterface.tsx

```diff
@@ -15,14 +15,47 @@ export interface JoinRoomState {
 
 export interface RoomDB {
     title: string;
-    roomUrl: string;
+    roomUrl: number | null;
+    roomID: number | null;
     users: User[];
     isNamePicked: boolean;
     rules: string;
-    dateOfExchange: string;
-    maxAmount: number | '';
+    dateOfExchange: string | null;
+    maxAmount: number | null;
+    pickedUser: PickedUserData | null;
 }
 
 interface User {
     user: string;
+}
+
+export interface RoomListDB {
+    rooms: RoomListDBElement[];
+}
+
+export interface RoomListDBElement {
```

Comment:
Do you export this one? If not you don't need export

Reply:
I do export this one, I think what you wanted to ask is if I use it outside this file and I do, `RoomList`, line 19. The export is necessary.